sig-docs/localization-subproject/README.md

```diff
@@ -0,0 +1,107 @@
+| Portuguese | Ricardo Katz ([rikatz](https://github.com/rikatz)), Mauren Berti ([@stormqueen1990](https://github.com/stormqueen1990)), Edson C ([@edsoncelio](https://github.com/edsoncelio)) | https://kubernetes.io/pt-br/ | [#kubernetes-docs-pt](https://kubernetes.slack.com/archives/CJ21AS0NA) | 
+| Russian | Dmitry Shurupov ([@shurup](https://github.com/shurup)) | https://kubernetes.io/ru/ | [#kubernetes-docs-ru](https://kubernetes.slack.com/archives/CPZ9KD9TN) |
+| Spanish | Rael Garcia ([@raelga](https://github.com/raelga)), Victor Morales ([@electrocucaracha](https://github.com/electrocucaracha/)) | https://kubernetes.io/es/ | [#kubernetes-docs-es](https://kubernetes.slack.com/archives/CH7GB2E3B) | 
+| Ukrainian | Maksym Vlasov ([@MaxymVlasov](https://github.com/MaxymVlasov)) | https://kubernetes.io/uk/ | [#kubernetes-docs-uk](https://kubernetes.slack.com/archives/CSKCYN138) | 
+| Vietnamese | | https://kubernetes.io/vi/ | [#kubernetes-docs-vi](https://kubernetes.slack.com/archives/CPHAWNF1Q) |  
+
+"In progress" teams have started localization work, but are not yet live on the Kubernetes site.
+
+### Localization team structure
+
+Anyone can start a new localization effort for any language or join an ongoing localization team. If you are interested in learning more about specific localization requirements or processes, see the [Localization Contributing Guide](https://kubernetes.io/docs/contribute/localization/).  
+
+Localization teams are self-sufficient communities within SIG Docs and are responsible for
+
+- Localizing content on kubernetes.io into their chosen language.  
+- Timely reviews and merging PRs for their languages according to the [review and merge processes](https://kubernetes.io/docs/contribute/review/) of the kubernetes/website repo.
+- Building a community that can support their localization efforts. Note that you need at least two contributors to begin a localization team because contributors can't approve their pull requests. You will likely need more contributors, reviewers, and approvers to create a sustainable localization effort.
+- Organizing and prioritizing work to localize content.
+- Understanding and following the [Kubernetes Code of Conduct](https://kubernetes.io/community/code-of-conduct/) within their teams. Questions or concerns around Code of Conduct violations should be raised to the SIG Docs, Localization Subproject leadership, or [Kubernetes Code of Conduct Committee](https://github.com/kubernetes/community/tree/master/committee-code-of-conduct). 
+
+The Localization Subproject leads and the SIG Docs leadership are always available to help localization teams by answering questions and providing feedback.
+
+### Localization team leads
+
+A leader of a localization team is someone who is responsible for helping to guide the localization efforts for a given language. Its recommended that each localization team has at least 2 leads.   
```

Comment:
>Do we need to guide `It's recommended that each localization team has at least 2 leads.` ?
(according to the above current list, only few teams satisfy the recommendation)

I think its still good to recommend that there are at least 2 people as leads, even if we aren't able to staff it. I think its good to still advocate for having more leaders because we don't want to have a single point of failure for a team if someone needs to step away from the project and a gentle nudge for leaders to keep looking for more folks to help them do the work Its worded as a "recommendation" and not a requirement for a localization team thought because its not always possible or available to folks. 

> we can make the team lead as an official role
I'd like to make the team lead more of an official role within the project so that folks can get the proper recognition. I don't know the process for formalizing a role though in the community but I think if this was to be a more formal role, folks who wanted to be a lead would need to go through the a nomination/approval process like in the other roles within the community.

Reply:
@seokho-son & @a-mccarthy : 

- Recommending 2 people as leads here is appropriate because it is not just a recommendation. At the moment, it is a requirement for new localisations going forward so that we can ensure consistency wrt contributions and not just when it goes live.
- Acknowledging team leads/localization maintainers is an idea I'm 100% on board with. However, these are the challenges I foresee with making that a reality, 
  - Processes aren't uniform across all localization. There will be concerns around the implementation if we go forward without achieving uniformity there first.
  - We'll need to address other major questions that come with the formalisation of a role like - maintaining staffing, a consistent contributor pool, & ensuring necessary diversity across the board. For some localisations that are relatively smaller, this might not be a discussion we can actually have.
  - Additionally, formalised roles would typically go through the nomination/approval process, seeking community consensus and would significantly increase the barrier to getting localisations started and live.

Reply:
Hi @a-mccarthy @divya-mohan0209,
I concur with the recommendations outlined in the section. Additionally, I acknowledge @divya-mohan0209 's concerns regarding the formalization of roles. My suggestion is that we move forward with the current terminology as it stands, specifically the recommendation that `each localization team should have at least two leads`. Subsequently, we can assess the practicality of formalizing these roles across the existing teams.